packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -95,7 +79,7 @@ class WalletsController {
     password: string
     mnemonic: string
   }): ChannelResponse<Wallet> => {
-    const storedKeystore = Key.fromMnemonic(mnemonic, true, password).getKeystoreString()
+    const storedKeystore = Key.fromMnemonic(mnemonic, password, 17, 3).getKeystore()
```

Comment:
What are the magic numbers `17` and `3`?

Reply:
17 and 3 come from 20 addresses in address page.

Maybe they can be set default value or other more better style.